### ISSUE TITLE
Dependencies for SNEWS_PT at IceCube

### ISSUE THREAD

**Issue by sgriswol** [SNEWS2/SNEWS_Publishing_Tools#42]
IceCube is in the process of upgrading the machines where SN alerts are handled. At the time of writing, the following packages are in conflict with SNEWS_PT's requirements. The highest package version currently available on this machine are also listed (in short, Python 3.6-compatible versions).
```
Package    SNEWS_PT Requires     Available Version
click      ~8.1.2                8.0.4
ipython    ~7.32.0               7.16.2
pandas     ~1.4.2                1.1.5
setuptools ~62.1.0               57.0.0
inquirer    2.9.1                2.8.0
```
This list is subject to change, so I will update this issue accordingly.

**Comment by sybenzvi** [comment]
Thanks for checking this @sgriswol. Question for @Storreslara and @KaraMelih: are the current SNEWS_PT requirements absolutely necessary, or can any of them be downgraded to older versions with no harm to functionality? I do suggest we investigate after the firedrill, since Spencer is making a workaround so we can participate in the test.

**Comment by KaraMelih** [comment]
Thanks for the feedback!
We were always assuming that the snews_pt would be installed on a fresh python environment thus any inconsistencies among the versions between the programs would be avoided. The snews_pt requires only a couple of string inputs or a JSON file and thus can interact with the detector within a virtual environment without any problem.
That being said, I think indeed some requirements are too tight and some are only needed for the development.
We can use larger e.g. `click > 8.0` instead of compatible e.g. `~8.1.2` flags, and remove the packages not needed for non-dev purposes such as sphinx related ones or `inquirer`.

From the version list above, _maybe_ pandas might cause problems depending on whether any feature we used was introduced before or after upgrading from `1.1.5` which should be easy to notice.

**Comment by Sheshuk** [comment]
@sgriswol, if you try installing SNEWS_PT under a new [venv](https://docs.python.org/3/library/venv.html), is it still unable to install the needed dependency packages versions?

**Comment by sgriswol** [comment]
@Sheshuk Yes, I am currently using a SNEW_PT-specific venv and am unable to install the needed dependencies on the machine we use to handle SNEWS alerts.  

To be more specific, the machine we are currently using is running SL 6.4, and currently uses OpenSSL 1.0.1. Click 8.1.2 requires Python 3.7 which we are installing from source, and doing so requires OpenSSL >=1.0.2. 

This machine is being upgraded to CentOS 7.9.2009, so once the upgrade is complete, we should be able to use more modern python and python package versions.

**Comment by joesmolsky** [comment]
Closing this since it is no longer an issue for IceCube and no one else seems to be using python 3.6.
